lib/portabilling/account_session_manager.ex

```diff
@@ -43,6 +43,11 @@ defmodule Portabilling.AccountSessionManager do
     GenServer.call(__MODULE__, {:get_session_id, i_account})
   end
 
+  @spec del_session_id(integer()) :: :ok
+  def del_session_id(i_account) do
+    GenServer.cast(__MODULE__, {:del_session_id, i_account})
```

Comment:
I propose to use the same `GenServer.call` notation as it was done for `get_session_id`.
Return `:ok` if the session id is removed for `i_account`, and `:error` if it is not.